src/services/RerouteService.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Reroute plugin for Craft CMS 3.x
+ *
+ * Manage 301/302 redirects in the control panel.
+ *
+ * @link      https://www.trevor-davis.com
+ * @copyright Copyright (c) 2018 Trevor Davis
+ */
+
+namespace davist11\reroute\services;
+
+use davist11\reroute\Reroute;
+
+use Craft;
+use craft\base\Component;
+use craft\db\Query;
+use davist11\reroute\models\RerouteModel;
+use davist11\reroute\records\RerouteRecord;
+
+/**
+ * @author    Trevor Davis
+ * @package   Reroute
+ * @since     2.0.0
+ */
+class RerouteService extends Component
+{
+    // Public Methods
+    // =========================================================================
+
+    /*
+     * @return mixed
+     */
+    public function getAll()
+    {
+        $reroutes = (new Query())
+            ->select(['id', 'oldUrl', 'newUrl', 'method'])
+            ->from('{{%reroute}}')
+            ->all();
```

Comment:
If you don't want to repeat the table name, you can change this to:

```
$reroutes = RerouteRecord::find()
            ->select(['id', 'oldUrl', 'newUrl', 'method'])
            ->all();
```

You already have `RerouteRecord` defined in this file